### PR TITLE
Added NK_DEFK17, NK_DT12, US_CCF12.json

### PR DESCRIPTION
**Pull request by JTWendelborn**
Deleted critical policy rules from selection of NK_DT12 and US_CCF12. Also added steady state file for NK_DT12. Now, everything should work fine.

### CODE REVIEW

**Comment by gboehl** [comment]
DEFK17 with Orphanides-Wieland works under Matlab but not under Octave. Can you have a look whats wrong here?

@j2L4e any clues?

The other models do fine, thanks for that. If you want, you can send a PR with those only or wait until we fixed DEFK17.

**Comment by JTWendelborn** [comment]
I'll check and send you a new PR ver the weekend.

**Comment by j2L4e** [comment]
@gboehl, both OW08 and OW13 work for me, but OW08 takes >30s to finish

File: static/mmci-cli/models/NK_DEFK17/NK_DEFK17.json [hunk 1]
```diff
@@ -0,0 +1,327 @@
...
+    {
+      "name": "ERQ_f",
+      "text": "ERQ_f"
+    },
+    {
+      "name": "w_f",
+      "text": "w_f"
+    },
+    {
+      "name": "infl_w",
```
Comment (j2L4e) [inline static/mmci-cli/models/NK_DEFK17/NK_DEFK17.json]: this is a duplicate of line 179

Comment (gboehl) [reply static/mmci-cli/models/NK_DEFK17/NK_DEFK17.json]: Is that a problem?

Comment (gboehl) [reply static/mmci-cli/models/NK_DEFK17/NK_DEFK17.json]: Otherwise I'd merge the PR...

Comment (j2L4e) [reply static/mmci-cli/models/NK_DEFK17/NK_DEFK17.json]: yes, it will lead to odd behavior of the variable seelction list.

Comment (JTWendelborn) [reply static/mmci-cli/models/NK_DEFK17/NK_DEFK17.json]: Sorry for that, must have happened when I added the flex-price variables. I'll correct that and send a new PR!

**Comment by JTWendelborn** [comment]
I corrected the mistake Jan pointed out. Also, as for Jan, OW13 and OW08 work for me under both, Matlab and Octave, but OW08 needs a bit longer under Octave.

**Pull request merged**